src/quo2/components/avatars/wallet_user_avatar.cljs

```diff
@@ -28,21 +28,26 @@
 (defn wallet-user-avatar
   "params, first name, last name, color, size
    and if it's dark or not!"
-  [{:keys [f-name l-name customization-color size]
+  [{:keys [f-name l-name customization-color size monospace? uppercase?]
     :or   {f-name              "John"
            l-name              "Doe"
            customization-color :red
-           size                :x-large}}]
+           size                :x-large
+           uppercase?          true}}]
   (let [circle-size    (size circle-sizes)
         small?         (= size :small)
         f-name-initial (-> f-name
-                           string/upper-case
+                           (#(if uppercase? (string/upper-case %) %))
                            (subs 0 1))
         l-name-initial (-> l-name
-                           string/upper-case
+                           (#(if uppercase? (string/upper-case %) %))
                            (subs 0 1))
-        circle-color   (colors/custom-color customization-color 50 20)
-        text-color     (colors/custom-color-by-theme customization-color 50 60)]
+        circle-color   (if (= customization-color :neutral)
```

Comment:
hmm to me this is a bit of a misuse of the customization-color prop. 
customization-color is something we want to pass directly to the resolver function - custom-color and let that do the handling of colors, so I think you should change this code a bit somehow. perhaps by some type or something 👍 
in fact I think we should just make this 
```
(if (= customization-color :neutral)
 

hmm to me this is a bit of a misuse of the customization-color prop. 
customization-color is something we want to pass directly to the resolver function - custom-color and let that do the handling of colors, so I think you should change this code a bit somehow. perhaps by some type or something 👍 
in fact I think we should just make this 
```
(if customization-color 
   (colors/custom-color customization-color 50 20)
   (colors/theme-colors colors/neutral-80-opa-5 colors/white-opa-5))
```
as that's something we pretty much do in all other components. 
btw @tumanov-alex  also has a pr to update this component so hopefully you are both aware of each other's pr's
https://github.com/status-im/status-mobile/pull/17328
Not sure if his pr caught that this variant existed.

Additionally (colors/custom-color-by-theme is removed in my pr here - https://github.com/status-im/status-mobile/pull/17567 so it's either a race :D or if you want to use colors/theme-colors & custom-color method then there won't be any clashes 👍

Reply:
Sure, I updated the implementation and removed the default customization color that was red for some reason. I know there are some color deprecated methods used but IMO when solving conflicts Alex's PR (as this change was solely to comply with Address List Item component design)

Reply:
Yep, sounds good 👌